<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddUser</string>
  </property>
  <widget class="QLabel" name="id_lab_user_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_passwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_confirm_passwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_permission_type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>权限级别：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_add_user">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新增用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_user_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_passwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_confirm_passwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="id_combox_permission_type">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_sure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
